test/app/views/integration_examples/render_component.html.erb

```diff
@@ -0,0 +1 @@
+<%= render_component ErbComponent.new(message: "bar") %>
```

Comment:
```suggestion
<%= render_component ErbComponent.new(message: "bar") %>

```

Reply:
Done in 10bdc944593c8b52f209630571c887df8e8a0bc8